k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -196,7 +201,7 @@
      * have already been added from the restore of the view state.
      */
     private boolean mSourceMessageProcessed = false;
-    private int mMaxLoaderId = 0;
+    private int mMaxLoaderId = 128;
```

Comment:
This one was a quick fix because the field is going to go away with #1392 anyways. You're right though, I should give it a better name here as well